develop-guide/writeblitz/writeblitz.rst

```diff
@@ -550,6 +569,41 @@ Action ``compare`` allows you to verify the values of the saved variables. Below
         - "'%VARIABLES{os}' == 'NX-OS' and '%VARIABLES{date_created}' == '10/22/2019 10:00:00 [10/22/2019 16:57:31]'"
         - " %VARIABLES{bootflash} >= 290000 or '%VARIABLES{bios}' == '07.33'"
 
+
+
+.. note::
+
+    The starting message of an step in etrade logs, is predefined. However, that value can be changed.
```

Comment:
trade is not released externally, cant say this.

The starting message of a Step can be modified by using ....

Reply:
addressed